lib/msal-common/src/account/ClientInfo.ts

```diff
@@ -32,3 +33,14 @@ export function buildClientInfo(rawClientInfo: string, crypto: ICrypto): ClientI
         throw ClientAuthError.createClientInfoDecodingError(e);
     }
 }
+
+export function buildClientInfoFromHomeAccountId(homeAccountId: string): ClientInfo {
+    const clientInfoParts: string[] = homeAccountId.split(Separators.CLIENT_INFO_SEPARATOR);
+    if (clientInfoParts.length < 2) {
+        throw ClientAuthError.createClientInfoDecodingError("Home account ID was malformed.");
```

Comment:
This may break ADFS use cases. Double check that we still include the separator even when there's no `utid`

Reply:
chatted offline, added case for when no `utid` is present (only `uid`)